models/ccsr/final/ccsr__long_condition_category.sql

```diff
@@ -1,5 +1,5 @@
 {{ config(
-     enabled = var('ccsr_enabled',var('claims_enabled',var('clinical_enabled',var('tuva_marts_enabled',False)))) | as_bool
```

Comment:
Just want to make sure this edit was intentional since we just added the as_bool to every model.

Reply:
It was not, thanks for catching that!  I've added it back in.